packages/appstore/src/components/wallet-cards-carousel/wallet-cards-carousel.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { getWalletHeaderButtons } from 'Constants/utils';
+import { TWalletAccount } from 'Types';
+import WalletButton from 'Components/wallet-button';
+import CardsSliderSwiper from './cards-slider-swiper';
+import { observer, useStore } from '@deriv/stores';
+import './wallet-cards-carousel.scss';
+
+type TProps = {
+    readonly items: TWalletAccount[];
+};
+
+const WalletCardsCarousel = observer(({ items }: TProps) => {
+    const {
+        client: { loginid, switchAccount },
+    } = useStore();
+
+    const [active_page, setActivePage] = React.useState(
```

Comment:
```suggestion
    const [active_wallet, setActiveWallet] = React.useState(
```

BTW is this changeable with @thisyahlen-deriv 's hook (useActiveWallet)?

Reply:
@mahdiyeh-deriv It's just for carousel pagination, because I'm using number of array of wallets when the user change the wallet, so I can't use here useActiveWallet hook because it returns just a wallet, but I need his number in the array of wallets